omnisafe/common/logger.py

```diff
@@ -23,12 +23,22 @@
 
 import numpy as np
 import torch
-from torch.utils.tensorboard import SummaryWriter
 
 from omnisafe.utils.distributed_utils import mpi_statistics_scalar, proc_id
 from omnisafe.utils.logger_utils import colorize, convert_json
 
 
+# As of torch v1.9.0, torch.utils.tensorboard has a bug that is exposed by setuptools 59.6.0.  The
+# bug is that it attempts to import distutils then access distutils.version without actually
+# importing distutils.version.  We can workaround this by prepopulating the distutils.version
+# submodule in the distutils module.
+# pylint: disable-next=wrong-import-order,unused-import
+import distutils.version  # isort:skip  # noqa: F401
```

Comment:
```suggestion
try:
	import distutils.version  # isort:skip  # noqa: F401
except ImportError:
    pass
```

`distutils` is officially deprecated by Python.